### PR TITLE
added working PKGBUILD 'activitywatch-bin'

### PR DESCRIPTION
**Pull request by ErikBjare**
Installs the PyInstaller builds into `/opt/activitywatch` and creates symlinks to the executables in `/usr/bin`.

### CODE REVIEW

**Comment by polyzen** [comment]
All the empty variables, arrays, and packaging functions are unnecessary.

File: scripts/package/arch-packages/activitywatch-bin/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+checkdepends=()
+optdepends=()
+provides=()
+conflicts=("activitywatch")
+replaces=()
+backup=()
+options=()
+install=
+changelog=
+source=("${pkgname}.zip::https://github.com/ActivityWatch/activitywatch/releases/download/v${pkgver}/activitywatch-v${pkgver}-linux-x86_64.zip")
```
Comment (polyzen) [inline scripts/package/arch-packages/activitywatch-bin/PKGBUILD]: No need to rename the source to `$pkgname.zip` here, but, more importantly, should keep the version in the filename.

**Comment by ErikBjare** [comment]
All that's left is to actually upload it to the AUR: https://wiki.archlinux.org/index.php/Arch_User_Repository#Sharing_and_maintaining_packages

**Comment by ErikBjare** [comment]
@polyzen Woah, quick comment!

File: scripts/package/arch-packages/activitywatch-bin/PKGBUILD (hunk 1 above)
Comment (ErikBjare) [reply scripts/package/arch-packages/activitywatch-bin/PKGBUILD]: Alright, so `$pkgname-$pkgver.zip`?

**Comment by ErikBjare** [comment]
Uploaded to the AUR: https://aur.archlinux.org/packages/activitywatch-bin/

**Pull request merged**